packages/web/components/swap-tool/index.tsx

```diff
@@ -90,17 +90,31 @@ export const SwapTool: FunctionComponent<SwapToolProps> = observer(
     const slippageConfig = useSlippageConfig();
 
     // out amount less slippage calculated from slippage config
-    const outAmountLessSlippage = useMemo(
-      () =>
+    const { outAmountLessSlippage, outFiatAmountLessSlippage } = useMemo(() => {
+      // Compute ratio of 1 - slippage
+      const oneMinusSlippage = new Dec(1).sub(slippageConfig.slippage.toDec());
+
+      // Compute out amount less slippage
+      const outAmountLessSlippage =
         swapState.quote && swapState.toAsset
-          ? new IntPretty(
-              swapState.quote.amount
-                .toDec()
-                .mul(new Dec(1).sub(slippageConfig.slippage.toDec()))
-            )
-          : undefined,
-      [swapState.quote, swapState.toAsset, slippageConfig.slippage]
-    );
+          ? new IntPretty(swapState.quote.amount.toDec().mul(oneMinusSlippage))
+          : undefined;
+
+      // Compute out fiat amount less slippage
+      const outFiatAmountLessSlippage = swapState.tokenOutFiatValue
```

Comment:
Thoughts on something like `outFiatValueLessSlippage`?

Removing the term "amount" and adding "value" would make the name a bit clearer IMO.

Reply:
Attempted the change. The suggested naming would break consistency with another related identifier `outAmountLessSlippage`

Renaming them all introduces confusion, to address which takes a lot of the diff. I think we should pursue this separately if this is desired

Reply:
That's the amount less the slippage. This is the fiat value less slippage. I'm not sure I'm following?

Reply:
`outFiatAmountLessSlippage` and `outAmountLessSlippage`

are 2 related identifiers so having a similar name improves readability. We always call `outAmount` as amount and not value. In fact, `outValue` feels more confusing.

For these reasons, I would prefer to keep the names as they are